mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1329,6 +1333,12 @@ not.
     * STABLE: Every few months upgrade cadence; Production users who need stability above all else, and for whom frequent upgrades are too risky.
     * EXTENDED: GKE provides extended support for Kubernetes minor versions through the Extended channel. With this channel, you can stay on a minor version for up to 24 months.
 
+<a name="nested_gke_auto_upgrade_config"></a>The `gke_auto_upgrade_config` block supports:
+
+* `patch_mode` - (Required) The selected patch mode.
```

Comment:
Should this be type not patch mode?

Reply:
I am confused.  gke_auto_upgrade_config.type is what is being proposed as a change.

https://github.com/GoogleCloudPlatform/magic-modules/pull/14265/files#diff-72f12c6607752854efe198a4fc418be5a02d5ba0964b3a41e232939db0575824R2102

This documentation does not reference type and references patch mode. This is documentation for the terraform fields. One or the other needs to change to have consistency. If what you are saying is that the API name is patch_mode then we should switch 'type' to that and vice versa.

Reply:
I think that line alone is a typo FWIW, it's patch_mode in the expander/flattener and everywhere else I saw: https://github.com/GoogleCloudPlatform/magic-modules/pull/14265/files#diff-72f12c6607752854efe198a4fc418be5a02d5ba0964b3a41e232939db0575824R5979